.changeset/hungry-cycles-drum.md

```diff
@@ -0,0 +1,6 @@
+---
+"@hashicorp/design-system-components": minor
+---
+
+`Hds::Pagination` - Converted component to Typescript
+- Converted to Typescript
```

Comment:
Oh, you can delete this line now.

Reply:
Whoops. Did this through the Github interface 😆